### ISSUE TITLE
New fields for overriding default illustration 

### ISSUE THREAD

**Issue by murtaugh** [alistapart/AListApart#114]
The current method of generating illustration images isn't sufficient now that we have multiple illustration styles.

We need a new set of fields allowing the art director to manually upload images, and the templates need to be updated to check for values in these fields before generating the default image variations.

@mikepick — can you confirm what those fields should be?

**Comment by mikepick** [comment]
My initial thinking is that we would want:

2x Large | 2X Small | Large | Small

and that would cover our use cases? @beep any thoughts?

As well, I believe out current system relies on images being the width of the container. It might be nice to leverage cropping as well in this setup.

**Comment by beep** [comment]
Hello! Thanks for looping me in, @mikepick!

Not sure I understand the contours of the problem, so please consider the below wonderfully ill-informed suggestions/thoughts.

> 2x Large | 2X Small | Large | Small

Are the 2× versions going to be, well, just double the size of their 1× counterparts? Could we just upload the retina/2× versions, and letting the system resize ’em down?

**Comment by murtaugh** [comment]
The thinking for the override fields is they are 100% manually created. Basically "the automated output looks derpy, so let me upload 4 versions that look good." So, the system doesn't get involved at that point (in terms of generating variations).

That said, if Mike simply didn't populate the 1x fields, we could have the system generate them, and them Mike could take the extra step of replacing those manually if he doesn't like the results.

**Comment by beep** [comment]
Nice, that makes sense. Could also adopt that for all four options, I suppose: if Mike supplied a manual fix, use that; otherwise, use the system default.

**Comment by michellekondou** [comment]
Is this working as expected? /cc @murtaugh

**Comment by murtaugh** [comment]
No, we haven't added the new fields yet. (There's a set of "Responsive Illustration" fields that are something completely different, which is maybe confusing.)
